en/mozillavpn.xliff

```diff
@@ -419,203 +427,237 @@
+      <source xml:space="preserve">View Logs</source>
+      <target xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">24</context></context-group>
+    </trans-unit>
+  </body></file>
   <file datatype="plaintext" original="../src/ui/views/ViewMain.qml" source-language="en" target-language="en-US"><body>
     <trans-unit id="MozillaVPN">
       <source xml:space="preserve">Mozilla VPN</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">92</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">90</context></context-group>
     </trans-unit>
     <trans-unit id="vpn.updates.newVersionAvailable">
       <source xml:space="preserve">New version is available.</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">120</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">107</context></context-group>
     </trans-unit>
     <trans-unit id="vpn.updates.updateNow">
       <source xml:space="preserve">Update now</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">122</context></context-group>
-      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewUpdate.qml</context><context context-type="linenumber">199</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">109</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewUpdate.qml</context><context context-type="linenumber">173</context></context-group>
     </trans-unit>
   </body></file>
   <file datatype="plaintext" original="../src/ui/views/ViewOnboarding.qml" source-language="en" target-language="en-US"><body>
     <trans-unit id="vpn.onboarding.skip">
       <source xml:space="preserve">Skip</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">63</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">66</context></context-group>
     </trans-unit>
     <trans-unit id="vpn.onboarding.next">
       <source xml:space="preserve">Next</source>
       <target xml:space="preserve" />
-      <context-group purpose="location"><context context-type="linenumber">72</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">75</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/views/ViewSubscriptionNeeded.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.subscription.title">
+      <source xml:space="preserve">Subscribe for %1/mo</source>
```

Comment:
Is "/mo" normal in English? Might be worth adding a note that it means "per month".

Reply:
I forgot: this one also needs a comment explaining what %1 is.